cli/server.go

```diff
@@ -719,7 +722,7 @@ func configureTLS(listener net.Listener, tlsMinVersion, tlsClientAuth, tlsCertFi
 	return tls.NewListener(listener, tlsConfig), nil
 }
 
-func configureGithubOAuth2(accessURL *url.URL, clientID, clientSecret string, allowSignups bool, allowOrgs []string) (*coderd.GithubOAuth2Config, error) {
+func configureGithubOAuth2(accessURL *url.URL, clientID, clientSecret string, allowSignups bool, allowOrgs []string, allowTeams []string) (*coderd.GithubOAuth2Config, error) {
```

Comment:
We check the `allowTeams` is formatted correctly when a user tries to login. So if the deployment has an issue, we don't know until login. Should we verify format up front in this function?

If the format is incorrect (not 2 parts), should we just log an error and kill coder from starting?

Reply:
On that note, should we pass this as a datastruct so we don't have to parse this again? That might be overkill, just would be unfortunate to have to strings.Split again and check for parts.

Reply:
I was hesitant about this at first, but I think it turned out cleaner this way. Good suggestion!